### DISCUSSION TITLE
Add references field to people JSON

### DISCUSSION THREAD

**Discussion by mauvehed** [restincode/restincode#176] [Ideas]
I think we need to add another section (like socialmedialinks, gallery, or contributions) that's for other miscellaneous things such as news articles about them, obituaries, stories, videos, etc. Things they didn't create, but relate to them.

**Comment by attritionorg** [comment]
@mauvehed and I chatted about this. I think a wikipedia heading would be good, even if they don't have a page. Maybe encourage someone to write one.

Obit we have I think. News articles are good as a separate category. All the socials yes. Then finally a generic 'References' section as a catch-all for the rest.
